<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainInputWidget</class>
 <widget class="QWidget" name="MainInputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <item>
    <layout class="QVBoxLayout" name="inputLayout" stretch="9,1">
     <item>
      <widget class="QGroupBox" name="inputGroupBox">
       <property name="title">
        <string>Input</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="exportLayout">
       <item>
        <widget class="QLabel" name="exportPathLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Export Path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="FileSelectionWidget" name="exportPathWidget" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
     <item>
      <widget class="QGroupBox" name="analysisGroupBox">
       <property name="title">
        <string>Analysis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="exportGroupBox">
       <property name="title">
        <string>GroupBox</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileSelectionWidget</class>
   <extends>QWidget</extends>
   <header>preppipe_gui_pyside6.componentwidgets.fileselectionwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
